ballet/update.py

```
@@ -0,0 +1,84 @@
+import funcy
+import git
+import json
+import os
+import tempfile
+import yaml
+
+from cookiecutter.main import cookiecutter
+from cookiecutter.prompt import prompt_for_config
+
+from ballet.compat import pathlib, safepath
+from ballet.util.log import logger
+from ballet.quickstart import generate_project
+
+PROJECT_CONTEXT_PATH = (
+    pathlib.Path(__file__).resolve().parent.joinpath('project_template', 'cookiecutter.json')
+)
+
+REPLAY_PATH = (
+    pathlib.Path.home().joinpath('.cookiecutter_replay', 'project_template.json')
+)
+
+def _create_replay(tempdir, context):
+    tempdir = pathlib.Path(tempdir)
+    slug = context['cookiecutter']['project_slug']
+    old_context = None
+    try:
+        # replace the current replay with our own
+        with open(REPLAY_PATH, 'r') as replay_file:
+            old_context = json.load(replay_file)
+        with open(REPLAY_PATH, 'w') as replay_file:
+            json.dump(context, replay_file)
+        generate_project(replay=True, output_dir=tempdir)
+    except:
```

dangerous bare except, perhaps `except KeyError:`